plugin/validator_test.go

```diff
@@ -2378,3 +2378,63 @@ func getErrForField(e error, path string) string {
 	}
 	return ""
 }
+
+func TestValidator_Transformations(t *testing.T) {
+	v, err := NewValidator(ValidatorOpts{})
+	assert.NoError(t, err)
+	schema, err := os.ReadFile("testdata/transformations.lua")
+	assert.NoError(t, err)
+	pluginName, err := v.LoadSchema(string(schema))
+	assert.EqualValues(t, "transformations", pluginName)
+	assert.NoError(t, err)
```

Comment:
These error assertions should use `require.NoError()` as it'll cause ill effects if these do error out & the test continues running.

Reply:
Didn't need to do all of them, but this works anyways, resolving 😄